test/spec/features/service/deploy_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'spec_helper'
+
+describe 'service deploy' do
+  it 'deploys a service' do
+    run("kontena service create test-1 redis:3.0")
```

Comment:
Why service is created with `run(...)` and other command with `kommando`? What's the difference?